.github/scripts/upload-github.sh

```diff
@@ -0,0 +1,38 @@
+#1/bin/sh
+
+set -eo pipefail
+
+# Upload the first json to the pytorch benchmark data repository
```

Comment:
what does 'first json' mean in this context? is this leftover from the time you ran 5 iter? or are there other nights' json files still in the dir, and you're grabbing the most recent one?

Reply:
No, one dir will only contain json files from the same nightly build. But if multiple iterations exist, I will only grab the first one. This remains true if the iteration number is 1, in which the first one is the only one.